fdbserver/ClusterController.actor.cpp

```diff
@@ -1692,20 +1692,37 @@ class ClusterControllerData {
 		if (req.configuration.regions.size() > 1) {
 			std::vector<RegionInfo> regions = req.configuration.regions;
 			if (regions[0].priority == regions[1].priority && regions[1].dcId == clusterControllerDcId.get()) {
+				TraceEvent("CCSwitchPrimaryDC", id)
+				    .detail("CCDcId", clusterControllerDcId.get())
+				    .detail("OldPrimaryDCId", regions[0].dcId)
```

Comment:
I missed some cases where the capitalization was inconsistent

```suggestion
				    .detail("OldPrimaryDcId", regions[0].dcId)
```